sdk/java/pom.xml

```
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.github.onsdigital</groupId>
+  <artifactId>dis-redirect-api-client-java</artifactId>
```

Wonder if this should be renamed to indicate the new location
```suggestion
  <artifactId>dis-redirect-api-sdk-java</artifactId>
```